azext_iot/tests/digitaltwins/test_dt_data_history_lifecycle_int.py

```diff
@@ -0,0 +1,485 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+from time import sleep
+from knack.log import get_logger
+import pytest
+from azext_iot.common.utility import unpack_msrest_error
+from azext_iot.tests.digitaltwins.dt_helpers import assert_system_data_attributes
+from . import DTLiveScenarioTest
+from . import (
+    ADX_RG,
+    ADX_CLUSTER,
+    ADX_DATABASE,
+    EP_RG,
+    EP_EVENTHUB_NAMESPACE,
+    EP_EVENTHUB_TOPIC,
+    generate_generic_id
+)
+
+logger = get_logger(__name__)
+ADH_REGION = "northeurope"
+enabled_data_history = os.environ.get("ADT_DH_PREVIEW_ENABLE")
+
+
+@pytest.mark.skipif(not enabled_data_history, "Wait until Data History Public Preview")
```

Comment:
`@pytest.mark.skipif(not enabled_data_history, reason="Wait until Data History Public Preview")`